Catappult/appcoins-wallet-android
# [feature/APPC-2466] New Bottom Navigation - Promotions

**What does this PR do?**

   Implements Promotions screen with the new architecture (also with single state!) and with the new bottom navigation.

   Note: Navigation from Home to Promotions is marked as TO DO, since home needs to be redone before actually implementing the navigation correctly. 
   Note (2): https://github.com/Catappult/appcoins-wallet-android/pull/118 must be accepted before this PR.

**Database changed?**

   No

**How should this be manually tested?**

  Every flow within Promotions must be tested to verify that it behaves as expected and as before. If you can, please also test that promotion related notifications also redirect correctly to Promotions screen (e.g. see PerkBonusAndGamificationService).

**What are the relevant tickets?**

  [APPC-2466](https://aptoide.atlassian.net/browse/APPC-2466)

**Code Review Checklist**
